spec/spec_helper.rb

```diff
@@ -0,0 +1,92 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```

Comment:
__[rubocop]__
* Line is too long. [92/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)